src/snowflake/snowpark/mock/mock_plan.py

```diff
@@ -4,13 +4,18 @@
 
 import importlib
 import inspect
+from enum import Enum
 from functools import cached_property, partial
-from typing import Dict, List, NoReturn, Optional, Union
+from typing import TYPE_CHECKING, Dict, List, NoReturn, Optional, Union
 from unittest.mock import MagicMock
 
+if TYPE_CHECKING:
+    from snowflake.snowpark.mock.mock_analyzer import MockAnalyzer
+
 import numpy as np
 import pandas as pd
 
+import snowflake.snowpark.mock.file_operation as mock_file_operation
```

Comment:
NIT: why not just directly import the functions?

Reply:
it's just to avoid confusion where the `put` and `read_file` are imported.